ament_xmllint/ament_xmllint/main.py

```diff
@@ -39,15 +39,21 @@ def main(argv=sys.argv[1:]):
         nargs='*',
         default=[os.curdir],
         help='The files or directories to check. For directories files ending '
-             'in %s will be considered.' %
-             ', '.join(["'.%s'" % e for e in extensions]))
+             'in %s will be considered. Unless overruled by the --extensions '
+             'argument' %
+             ', '.join(["'.%s'" % e for e in default_extensions]))
     parser.add_argument(
         '--exclude',
         nargs='*',
         default=[],
         help='Exclude specific file names and directory names from the check')
     # not using a file handle directly
     # in order to prevent leaving an empty file when something fails early
+    parser.add_argument(
+        '--extensions',
+        nargs='*',
+        default=default_extensions,
```

Comment:
```suggestion
        default=['xml'],
```

(and then we can remove the line of code at the beginning of this function)

Reply:
No, as `default_extensions` is also used to generate the help of the `paths` option.

I don't think we want to maintain that list in two spots. Or we remove the extensions from that help msg.